corehq/ex-submodules/casexml/apps/phone/exceptions.py

```diff
@@ -39,6 +39,10 @@ class InvalidSyncLogException(Exception):
     pass
 
 
+class MissingSyncLogSQLException(Exception):
```

Comment:
Is it necessary to have a different exception? Can't we just use the `MissingSyncLog` exception?

Reply:
Yeah, I think we could, I will check